test/integration/connectors/test_neo4j.py

```diff
@@ -0,0 +1,215 @@
+import json
+import time
+import uuid
+from datetime import datetime
+from pathlib import Path
+
+import pytest
+from neo4j import AsyncGraphDatabase, Driver, GraphDatabase
+from neo4j.exceptions import ServiceUnavailable
+from pytest_check import check
+
+from test.integration.connectors.utils.constants import DESTINATION_TAG, env_setup_path
+from test.integration.connectors.utils.docker_compose import docker_compose_context
+from unstructured_ingest.error import DestinationConnectionError
+from unstructured_ingest.v2.interfaces.file_data import (
+    FileData,
+    FileDataSourceMetadata,
+    SourceIdentifiers,
+)
+from unstructured_ingest.v2.processes.connectors.neo4j import (
+    Label,
+    Neo4jAccessConfig,
+    Neo4jConnectionConfig,
+    Neo4jUploader,
+    Neo4jUploaderConfig,
+    Neo4jUploadStager,
+    Relationship,
+)
+
+USERNAME = "neo4j"
+PASSWORD = "password"
+URI = "neo4j://localhost:7687"
+DATABASE = "neo4j"
+
+EXPECTED_ELEMENT_COUNT = 42
+EXPECTED_CHUNKS_COUNT = 22
+EXPECTED_DOCUMENT_COUNT = 1
+EXPECTED_NODES_COUNT = 65
+DOCKER_COMPOSE_FILEPATH = env_setup_path / "neo4j" / "docker-compose.yml"
+
+
+# NOTE: Precheck tests are read-only so we utilize the same container for all tests.
+# If new tests require clean neo4j container, this fixture's scope should be adjusted.
+@pytest.fixture(autouse=True, scope="module")
+def _neo4j_server():
+    with docker_compose_context(DOCKER_COMPOSE_FILEPATH):
```

Comment:
Since you only need one container, this would be better using the docker utility rather than the docker compose one: [docker.py](https://github.com/Unstructured-IO/unstructured-ingest/blob/main/test/integration/connectors/utils/docker.py#L107). There are plenty of examples of that being used.

Reply:
I've replaced it with `container_context` and removed the compose file and directory.